src/features/MovieList/styled.js

```diff
@@ -1,25 +1,33 @@
 import styled from "styled-components";
 
 export const Content = styled.section`
-    max-width: 1368px;
-    margin: auto;
+    max-width: 1400px;
+    padding:0 16px;
+    margin: 0 auto;
     display: flex;
     flex-direction: column;
 `;
 
-export const Title = styled.h1`
-    font-family: 'Poppins';
-    font-style: normal;
-    font-weight: 600;
+export const Title = styled.h2`
     font-size: 36px;
-    line-height: 120%;
     margin: 0;
-    padding: 56px 16px 24px 16px;
+    padding: 56px 0 24px;
+
+    @media (max-width: 1023px ) {
+        font-size:3.5vw;
+    }
+
+    @media (max-width: 500px) {
+        font-size:17.3px;
+    }
 `;
 
 export const Wrapper = styled.div`
     display: grid;
```

Comment:
Consider changing this div to `<ul>` and MovieTile component to `<li>`. What we are presenting in a MovieList component is in fact a list of movies, an unordered one in this case. 

[Some Informations about HTML lists](https://www.w3.org/TR/html401/struct/lists.html#h-10.1)

Reply:
okey it makes sense i will change this

Reply:
okay I understood now, thanks